solr/solrj/src/test/org/apache/solr/client/solrj/impl/Http2SolrClientTest.java

```diff
@@ -1013,7 +1013,8 @@ public void testGetRawStream() throws Exception {
                 getBaseUrl() + "/debug/foo", DEFAULT_CONNECTION_TIMEOUT, DEFAULT_CONNECTION_TIMEOUT)
             .build()) {
       GenericSolrRequest req =
-          new GenericSolrRequest(SolrRequest.METHOD.GET, "/select", params("q", "*:*"));
+          new GenericSolrRequest(SolrRequest.METHOD.GET, "/select", params("q", "*:*"))
```

Comment:
Could this be replaced with a `QueryRequest` instead?

Reply:
I didn't try it earlier because I wasn't sure whether this was related to any of the `/debug/foo` stuff above, but I'll give it a shot and update if the test passes...

Reply:
Hey it worked - good catch!

Reply:
yay!